features/api.feature

```diff
@@ -129,6 +168,20 @@ Feature: DrupalContext
     Then I should see "Tag one"
     And I should see "Tag two"
 
+  @d8
+  Scenario: Create terms using vocabulary title rather than machine name.
+    Given "Tags" terms:
+    | name    |
+    | Tag one |
+    | Tag two |
+    And I am logged in as a user with the "administrator" role
+    When I go to "admin/structure/taxonomy/manage/tags/overview"
+    Then I should see "Tag one"
+    And I should see "Tag two"
+
+  @d7 @d8wip
+  # TODO: This doesn't work on Drupal 8 yet. For nodes the 'author' field is
+  # called 'uid' and only accepts numerical IDs.
   Scenario: Create nodes with specific authorship
     Given users:
     | name     | mail            | status |
```

Comment:
In order to create a node with a specific authors on Drupal 8 now we need to know the UID. We should make a followup for this. I guess this should be added to DrupalDriver: if the 'author' field is given when creating a node, convert this to the UID behind the scenes.

This is currently the only test that is not working for Drupal 8, all the others are running now.